src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import duke.commands.Command;
+import duke.exceptions.DukeException;
+import duke.exceptions.NoSuchCommandException;
+import duke.exceptions.NoSuchTaskException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
```

Comment:
i recommend to list the imports from java... before those from duke..., since it follows check style.